dali/kernels/imgproc/jpeg/jpeg_distortion_gpu_test.cu

```diff
@@ -37,7 +37,7 @@ class JpegDistortionTestGPU : public ::testing::Test {
   static constexpr bool horz_subsample = GTestParams::horz_subsample;
 
  public:
-  JpegDistortionTestGPU() {
+  JpegDistortionTestGPU() : input_device_(nullptr), output_(nullptr) {
```

Comment:
As a rule, it's preferable to initialize members in declaration instead of constructor - this way every new constructor doesn't have to repeat the initialization.
Also, this file is under active development by @jantonguirao, so changing it here is a disfavor.

Reply:
This is [WIP] to test Coverity and see what it reports and how. Please wait with the reviews.